src/Main.hs

```diff
@@ -0,0 +1,20 @@
+import System.Environment (getArgs)
+
+import Month (monthString)
+import Year (yearString)
+
+main :: IO ()
+main = do
+  args <- map read <$> getArgs -- TODO: read will explode on non-numeric input
```

Comment:
The types:
```haskell
getArgs :: IO [String]
map read :: Read a => [String] -> [a]
(<$>) :: Functor f => (a -> b) -> f a -> f b -- infix version of fmap
```

If we specialize `Read` to `Int` and `Functor` to `IO` and then substitute, we get:
```haskell
([String] -> [Int]) -> IO [String] -> IO [Int]
--  map read ^         getArgs ^      result ^
```

Read up on the `Functor` chapter in the Haskell Book for more info.